### PR TITLE
4b add testsummary

### PR DESCRIPTION
**Pull request by Amoondria**
Populated ResultSummary & added to Coordinator
Added TestCreatorParameters class
Made sure we're returning the (test) outcome as bool

### CODE REVIEW

File: 4b/Grains/src/Test1.cs [hunk 1]
```diff
@@ -11,19 +9,22 @@ public class Test1 : Orleans.Grain, ITest1
...
-            executionTimeDetails.Duration = stopwatch.Elapsed.ToString();
-            
-            var unitTest = Helpers.UnitTestCreator(this.GetType(), Helpers.CallerName(), testListId, executionTimeDetails);
+            var unitTest = Helpers.TestDetailsCreator(new TestCreatorParameters()
+            {
+                ClassType = GetType(),
+                CallerName = Helpers.CallerName(),
+                TestListId = testListId,
+                StartTime = startTime.ToString("yyyy-MM-ddThh:mm:ss.fff"),
+                EndTime = endTime.ToString("yyyy-MM-ddThh:mm:ss.fff"),
```
Comment (PiotrJustyna) [inline 4b/Grains/src/Test1.cs:23]: Was this format not supposed to be a constant string?

File: 4b/Grains/src/Test2.cs [hunk 2]
```diff
@@ -11,20 +9,23 @@ public class Test2 : Orleans.Grain, ITest2
...
+            var endTime = DateTime.UtcNow;
+            var duration = endTime - startTime;
             
+            var unitTest = Helpers.TestDetailsCreator(new TestCreatorParameters()
+            {
+                ClassType = GetType(),
+                CallerName = Helpers.CallerName(),
+                TestListId = testlistId,
+                StartTime = startTime.ToString("yyyy-MM-ddThh:mm:ss.fff"),
+                EndTime = endTime.ToString("yyyy-MM-ddThh:mm:ss.fff"),
```
Comment (PiotrJustyna) [inline 4b/Grains/src/Test2.cs:23]: Same here.

**Review by PiotrJustyna — APPROVED** [review]
Very nice, that will be a great starting point for when we can resume this. 🌟

File: 4b/Grains/src/Test1.cs (hunk 1 above)
Comment (Amoondria) [reply 4b/Grains/src/Test1.cs:23]: Oh! One of us can extract these first thing when we go back to working on these. I'll leave a comment on my local branch. Thanks for pointing it out :)

File: 4b/Grains/src/Test2.cs (hunk 2 above)
Comment (Amoondria) [reply 4b/Grains/src/Test2.cs:23]: Same thing as above 👍

**Pull request merged**